mobile/lib/db/upload_locks_db.dart

```diff
@@ -291,6 +307,67 @@ class UploadLocksDB {
     );
   }
 
+  Future<void> createStreamEntry(
+    int uploadedFileID,
+    String errorMessage,
+  ) async {
+    final db = await UploadLocksDB.instance.database;
+
+    await db.insert(
+      _streamUploadErrorTable.table,
+      {
+        _streamUploadErrorTable.columnUploadedFileID: uploadedFileID,
+        _streamUploadErrorTable.columnErrorMessage: errorMessage,
+        _streamUploadErrorTable.columnLastAttemptedAt:
+            DateTime.now().millisecondsSinceEpoch,
+      },
+    );
```

Comment:
Let's specify the conflict resolution here. insertOrReplace?

Reply:
insertOrReplace is not a method.